test/jdk/java/awt/PrintJob/PrintCheckboxTest/PrintCheckboxManualTest.java

```diff
@@ -22,84 +22,50 @@
  */
 
 /*
-  @test
-  @bug 5045936 5055171
-  @summary Tests that there is no ClassCastException thrown in printing
-   checkbox and scrollbar with XAWT
-  @key printer
-  @run applet/manual=yesno PrintCheckboxManualTest.html
+ * @test
+ * @bug 5045936 5055171
+ * @summary Tests that there is no ClassCastException thrown in printing
+ *          checkbox and scrollbar with XAWT
+ * @key printer
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual PrintCheckboxManualTest
 */
 
-// Note there is no @ in front of test above.  This is so that the
-//  harness will not mistake this file as a test file.  It should
-//  only see the html file as a test file. (the harness runs all
-//  valid test files, so it would run this test twice if this file
-//  were valid as well as the html file.)
-// Also, note the area= after Your Name in the author tag.  Here, you
-//  should put which functional area the test falls in.  See the
-//  AWT-core home page -> test areas and/or -> AWT team  for a list of
-//  areas.
-
-
-
-import java.awt.*;
-import java.awt.event.*;
-
-
-//Manual tests should run as applet tests if possible because they
-// get their environments cleaned up, including AWT threads, any
-// test created threads, and any system resources used by the test
-// such as file descriptors.  (This is normally not a problem as
-// main tests usually run in a separate VM, however on some platforms
-// such as the Mac, separate VMs are not possible and non-applet
-// tests will cause problems).  Also, you don't have to worry about
-// synchronisation stuff in Applet tests the way you do in main
-// tests...
-
-
-public class PrintCheckboxManualTest extends Panel
-{
-    //Declare things used in the test, like buttons and labels here
-    Frame f;
-
-    public static void main(String[] args) {
-        PrintCheckboxManualTest a = new PrintCheckboxManualTest();
-
-        a.init();
-        a.start();
+import java.awt.Button;
+import java.awt.Checkbox;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.GridLayout;
+import java.awt.Panel;
+import java.awt.PrintJob;
+import java.awt.Scrollbar;
+import java.awt.Toolkit;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class PrintCheckboxManualTest extends Panel {
+
+    private static final String INSTRUCTIONS = """
+            1. Click the 'Print' button on the frame
+            2. Select a printer in the print dialog and proceed
+            3. If the frame with checkbox and button on it is printed successfully test PASSED else FAILED.
```

Comment:
The frame decorations look different between the printed and on-screen versions.

![image](https://github.com/user-attachments/assets/8af0383b-a0e9-4f45-9343-161052026aab)